app/frontend/styles/_postcode.scss

```diff
@@ -0,0 +1,3 @@
+.govuk-input[autocomplete="postal-code"] {
```

Comment:
I would target an `#id` instead 👍

Reply:
I've changed my mind on this. Actually, targeting by the autocomplete attribute is good.